wallet-webapps/src/main/webapp/vue-app/components/wallet-balance/WalletBalance.vue

```diff
@@ -19,7 +19,10 @@
               class="white">
               <v-flex d-flex xs12>
                 <v-card flat>
-                  <v-card-text class="subtitle-2 grey--text pa-2">{{ this.$t('exoplatform.wallet.title.walletBalanceTitle') }}</v-card-text>
+                  <v-card-text
+                    class="subtitle-2 grey--text pa-2">
```

Comment:
please avoid using Vuetify colors helpers and use eXo colors helpers to make it more `brandable`. Thanks.

Reply:
Done!